### ISSUE TITLE
Uncertainties and variances

### ISSUE THREAD

**Issue by m-wells** [davidwhogg/DataAnalysisRecipes#42]
I believe there is a typo in https://github.com/davidwhogg/DataAnalysisRecipes/blob/710a6c959aa0072607d82e9a67a2f389a4c943d3/straightline/straightline.tex#L370 which should be
```latex
    When the uncertainties are Gaussian and their variances $\sigma_{yi}^2$
```
I also found this problem to be poorly worded https://github.com/davidwhogg/DataAnalysisRecipes/blob/710a6c959aa0072607d82e9a67a2f389a4c943d3/straightline/straightline.tex#L1767-L1774

Having done this problem, I can attest that the figures are made such that the uncertainties are set to _S_ not the variances.
However the statement
```latex
setting all $\sigma_{yi}^2=S$, that is, ignoring the uncertainties and replacing them all with the same value $S$
```
is not self-consistent.
I believe it should read as
```latex
\begin{problem}\label{prob:chi2} 
 Re-do the fit of \problemname~\ref{prob:easy} but setting all 
 $\sigma_{yi}=S$, that is, ignoring the uncertainties and replacing 
 them all with the same value $S$.  What uncertainty $S$ would 
 make $\chi^2 = N-2$?  Relevant plots are shown in 
 \figurename~\ref{fig:chi2}.  How does it compare to the mean and 
 median of the uncertainty $\allsigmay$? 
 \end{problem} 
```

**Comment by jobovy** [comment]
I don't think this is correct that we set sigma_y equal to S in this exercise (exercise 11 in the arXiv version). The code that produces these figures is [here](https://github.com/davidwhogg/DataAnalysisRecipes/blob/710a6c959aa0072607d82e9a67a2f389a4c943d3/straightline/src/ex9.py) (we re-numbered the exercises after writing the code) and you can see that [we set the data covariance to S x identity matrix](https://github.com/davidwhogg/DataAnalysisRecipes/blob/710a6c959aa0072607d82e9a67a2f389a4c943d3/straightline/src/ex9.py#L71) as stated in the problem.

**Comment by m-wells** [comment]
@jobovy You are correct and I was in mind edit correction as I was looking my work. I still think the wording is a bit unclear though.

**Comment by m-wells** [comment]
@jobovy I have edited my initial comment.

**Comment by m-wells** [comment]
My confusion essentially boils down to two typos which I have fixed in the PR.
